templates/landscape/managed.html

```diff
@@ -15,250 +17,272 @@
-                  </thead>
-                  <tbody>
-                    <tr>
-                      <th>Managed Landscape on public clouds, VMware, OpenStack or LXD</th>
-                      <td>$3,099</td>
-                      <td>$6,861</td>
-                    </tr>
-                    <tr>
-                      <th>Managed Landscape on bare metal</th>
-                      <td>N/A</td>
-                      <td>$9,470</td>
-                    </tr>
-                  </tbody>
-                </table>
+              <div class="p-logo-section__item">
+                {{ image(url="https://assets.ubuntu.com/v1/bd83cd83-opensearch-logo.png",
+                                alt="OpenStack",
+                                width="81",
+                                height="209",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
+              </div>
+              <div class="p-logo-section__item">
+                {{ image(url="https://assets.ubuntu.com/v1/7589afda-vmware-logo.png",
+                                alt="VMWare",
+                                width="216",
+                                height="209",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
               </div>
             </div>
           </div>
         </div>
-      </section>
+      </div>
+    </div>
+  </section>
 
-      <section class="p-strip is-white" style="background: white;">
-        <div class="row">
-          <div class="col-9 col-start-large-4 col-medium-4 col-start-medium-3 p-strip">
-            <h2>Got questions?</h2>
-            <p>Landscape's API can bridge any requirements our dashboard may not natively address.</p>
-            <p>
-              <a href="/landscape/managed#get-in-touch">Contact us about your unique requirements</a>
-            </p>
-          </div>
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <div class="col">
+          <h2>Bring your own identity and access management platform</h2>
+        </div>
+        <div class="col">
+          <p>
+            Landscape supports OpenID Connect (OIDC) to authenticate users, and integrates enterprise identity providers like Azure AD, Google, Okta, IBM, and more.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="u-fixed-width u-hide--small">
+      {{ image(url="https://assets.ubuntu.com/v1/9423d6a0-bring-own-identity.png",
+            alt="",
+            width="2464",
+            height="1028",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-image-container--full-width is-highlighted"}) | safe
```

Comment:
Image template needs formatting

Reply:
Could you please double check? I believe its formatted correctly.
Also, This is formatted with djlint, and changing it in any way reverts it back to this formatting automatically when formatting the doc.

Reply:
![image](https://github.com/user-attachments/assets/b1a274a8-417c-4098-b2b3-3e19510c70af)
Generally we have it like this
![image](https://github.com/user-attachments/assets/7bd0cee7-b949-45a5-bfcb-461448e151c9)